### PR TITLE
finished milestone 1

### PR DESCRIPTION
**Pull request by nnayk**
# Milestone: 1
# Developer: Nakul Nayak

### Pull Request Summary
This is my recipe site for milestone 1.

{Describe the purpose of your pull request}

### Pull Request Checklist
- [ X] Code is neat, readable, and works
- [ X] Comments are appropriate
- [X ] The commit message follows our guidelines
- [ X] The milestone number is specified
- [ X] The developer name is specified
- [ X] The summary is completed

### Reviewer Checklist  - PULL REQUEST REVIEWER ONLY

IMPORTANT: The rest of the sections in this checklist should only be filled out by authorized pull request reviewers. If you are the individual template contributor, do not fill out the rest of the fields or check the boxes.

NOTE: Milestones can only be completed when all boxes are checked.

- [x] The code is fully reviewed
- [x] Meaningful feedback is given
- [x] Comment that you have reviewed the code
- [x] This box is checked

### CODE REVIEW

**Comment by vwinstea** [comment]
Make sure to add all of your new files before committing them, right now you have no files committed. It looks like you have another repo nested inside your milestone repo. This stackoverflow page may help: [https://stackoverflow.com/questions/62056294/github-folders-have-a-white-arrow-on-them](https://stackoverflow.com/questions/62056294/github-folders-have-a-white-arrow-on-them). Please reach out if you have any questions!

**Comment by nnayk** [comment]
sorry about that, I've pushed the files now.

File: about.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+.header
+{
+    display: flex;
+    justify-content: space-between;
+    color: white;
+    /* align-items: center; */
+    margin: 5px;
+    /* margin-bottom: 10px; */
+    /* justify-content: space-around; */
```
Comment (vwinstea) [inline about.css]: Delete commented out rules. If you were having issues with flexbox here just lmk and I'll see if I can help!

File: about.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About me</title>
+    <link rel="stylesheet" href="about.css">
+</head>
+<body>
+    <div class="header">
+        <p class="logo"><strong></strong></p>
```
Comment (vwinstea) [inline about.html]: Let me know if you need help positioning a logo!

File: index.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+<body>
+    <div class="content">
+    <div class="block-one">
+        <div class="header">
+            <p class="logo"><strong></strong></p>
+            <ul class="header-links">
+                <li><a href="#">Home</a></li>
+                <li><a href="./about.html">About</a></li>
+            </ul>
+        </div>
```
Comment (vwinstea) [inline index.html]: Use the `<header>` tag to denote where the header is instead of the `<div>` you are currently using

Comment (vwinstea) [reply index.html]: Also, you might want to move your navbar out of the "block-one" div so the navbar looks the same across all your pages. Right now, it's positioned differently on your homepage

File: biryani.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    <div class="header">
+        <p class="logo"><strong></strong></p>
+        <ul class="header-links">
+            <li><a href="./index.html">Home</a></li>
+            <li><a href="./about.html">About</a></li>
+        </ul>
+    </div>
+    <h1>Chicken Biryani</h1>
+    <p>This is a popular Indian dish consisting of spicy, flavored rice and chicken.</p>
+    <h2>Ingredients</h2>
```
Comment (vwinstea) [inline biryani.html]: Recipe pages should include an image

**Review by vwinstea — CHANGES_REQUESTED** [review]
Good progress, but there are a few things I think you should change before porting wrapping up this milestone:

- [ ] Add more CSS to your about page and recipe page: more color, different fonts from the default, and adjusting the positioning between some content are all good first steps
- [x] Some pages are missing content: the homepage should include recipe descriptions and each recipe page should have an image
- [x] I left some comments on places in your code where things were commented out or not doing anything. Please reach out if you want help fixing those, otherwise just delete the commented out code

Please reach out on this PR or via slack if you have any questions or concerns. Good luck wrapping up this milestone!

**Comment by nnayk** [comment]
I've been quite busy this quarter so apologies for the delays...I updated the code based on the feedback. I wasn't able to fix the header issue in this milestone but I was able to do so for milestone 3

**Comment by nnayk** [comment]
Since my milestone 1 pull request is still pending, I'm not sure how to access the pull request for milestone 3 right now.

**Review by vwinstea — COMMENTED** [review]
Great work, thanks for getting those changes in! It's looking a lot better between the added content, cleaned up code, and added styling. Moving forwards, I would recommend pushing yourself a bit more on CSS, especially with more flexbox/layout stuff. The CSS is fine for this milestone though. Overall it's looking good though!

**Comment by vwinstea** [comment]
Milestone 1 complete! You should be able to make a new PR for milestone 3 now.

**Pull request closed without merging**